TestModels/aws-sdks/kms/test/TestComAmazonawsKms.dfy

```diff
@@ -100,6 +113,22 @@ module TestComAmazonawsKms {
     expect KeyId.value == expectedKeyId;
   }
 
+  method BasicDecryptTestFailureCase(
```

Comment:
Just adding one failure case for now. We should add more failure case when we (everyone who maintains smithy dafny) have more time.